### DISCUSSION TITLE
audio bug ? or screen mirroring bug

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#311] [Q&A] [answered]
hi ! every time im animating on stream, the audio in my project crackles and buffers ? D: idk if its cause the screen mirroring (it might possibly be, because when i tested the project without screen mirroring, it wasn’t crackling at all)

no clue if its a toonsquid issue or S.M issue (might be tbh) but just wanted to be sure D: !

**Comment by Cat-Butler** [comment, marked as answer]
I'm wondering if this is not a bug as such, but instead shows your CPU/GPU struggling to keep up with updating both screens and playing audio, all at the same time.

**Reply by trinhpxl** [reply]
mayhaps ! i’ll check my gpu usage next time when streaming and all :0 thank u for the possible answer !

**Reply by trinhpxl** [reply]
turns out it is my cpu usage, it says its high. so is there any way u know that can fix the crackling or delay ? :0 or do i just deal with it lmao xD ?

**Reply by keiwando** [reply]
I've noted it down and will look into it in the future. Thanks for the report!
